scss/_variables.scss

```diff
@@ -1,7 +1,7 @@
 $theme-colors: (
   "primary": #1abc9c,
   "secondary": #009b9e,
-  "success": #273647,
+  "success": #06744f,
```

Comment:
@lauggh @StefanoMiC Is the color change ok?

Reply:
I reverted this change, I thought the success was too dark but i'll leave that for another issue